### DISCUSSION TITLE
ReadMe

### DISCUSSION THREAD

**Discussion by Rootie2shoes** [community/community#19957] [Other Feature Feedback, Questions, & Ideas]
This XML file does not appear to have any style information associated with it. The document tree is shown below.

<feed xmlns="http://www.w3.org/2005/Atom">

<title type="text" xml:lang="en">Visual Studio Code - Code Editing. Redefined.</title>

<link type="application/atom+xml" href="https://code.visualstudio.com/feed.xml" rel="self"/>

<link type="text/html" href="https://code.visualstudio.com/" rel="alternate"/>

<updated>2022-06-09T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/</id>

<rights>Copyright (c) 2021, Microsoft, Inc.; all rights reserved.</rights>

<entry>

<title>Visual Studio Code May 2022</title>

<link href="https://code.visualstudio.com/updates/v1_68"/>

<updated>2022-06-09T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_68</id>

<content type="html"> <p>Visual Studio Code May 2022</p> <p><a href="https://code.visualstudio.com/updates/v1_68">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Development Container CLI</title>

<link href="https://code.visualstudio.com/blogs/2022/05/18/dev-container-cli"/>

<updated>2022-05-18T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2022/05/18/dev-container-cli</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2022/05/18/dev-container-cli">Read the full article</a></p> </content>

<author>

<name>Brigit Murtaugh</name>

</author>

</entry>

<entry>

<title>Visual Studio Code April 2022</title>

<link href="https://code.visualstudio.com/updates/v1_67"/>

<updated>2022-05-05T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_67</id>

<content type="html"> <p>Visual Studio Code April 2022</p> <p><a href="https://code.visualstudio.com/updates/v1_67">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Using Containers to move from local to Remote Development</title>

<link href="https://code.visualstudio.com/blogs/2022/04/04/increase-productivity-with-containers"/>

<updated>2022-04-04T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2022/04/04/increase-productivity-with-containers</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2022/04/04/increase-productivity-with-containers">Read the full article</a></p> </content>

<author>

<name>Olivia Guzzardo</name>

</author>

</entry>

<entry>

<title>Visual Studio Code March 2022</title>

<link href="https://code.visualstudio.com/updates/v1_66"/>

<updated>2022-03-30T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_66</id>

<content type="html"> <p>Visual Studio Code March 2022</p> <p><a href="https://code.visualstudio.com/updates/v1_66">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>The problem with tutorials</title>

<link href="https://code.visualstudio.com/blogs/2022/03/08/the-tutorial-problem"/>

<updated>2022-03-08T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2022/03/08/the-tutorial-problem</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2022/03/08/the-tutorial-problem">Read the full article</a></p> </content>

<author>

<name>Burke Holland</name>

</author>

</entry>

<entry>

<title>Visual Studio Code February 2022</title>

<link href="https://code.visualstudio.com/updates/v1_65"/>

<updated>2022-03-03T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_65</id>

<content type="html"> <p>Visual Studio Code February 2022</p> <p><a href="https://code.visualstudio.com/updates/v1_65">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Visual Studio Code January 2022</title>

<link href="https://code.visualstudio.com/updates/v1_64"/>

<updated>2022-02-03T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_64</id>

<content type="html"> <p>Visual Studio Code January 2022</p> <p><a href="https://code.visualstudio.com/updates/v1_64">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Visual Studio Code November 2021</title>

<link href="https://code.visualstudio.com/updates/v1_63"/>

<updated>2021-12-08T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_63</id>

<content type="html"> <p>Visual Studio Code November 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_63">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Notebooks, Visual Studio Code style</title>

<link href="https://code.visualstudio.com/blogs/2021/11/08/custom-notebooks"/>

<updated>2021-11-08T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/11/08/custom-notebooks</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/11/08/custom-notebooks">Read the full article</a></p> </content>

<author>

<name>Tanha Kabir</name>

</author>

</entry>

<entry>

<title>Visual Studio Code October 2021</title>

<link href="https://code.visualstudio.com/updates/v1_62"/>

<updated>2021-11-04T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_62</id>

<content type="html"> <p>Visual Studio Code October 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_62">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>vscode.dev Visual Studio Code for the Web</title>

<link href="https://code.visualstudio.com/blogs/2021/10/20/vscode-dev"/>

<updated>2021-10-20T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/10/20/vscode-dev</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/10/20/vscode-dev">Read the full article</a></p> </content>

<author>

<name>Chris Dias</name>

</author>

</entry>

<entry>

<title>Webview UI Toolkit for Visual Studio Code</title>

<link href="https://code.visualstudio.com/blogs/2021/10/11/webview-ui-toolkit"/>

<updated>2021-10-11T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/10/11/webview-ui-toolkit</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/10/11/webview-ui-toolkit">Read the full article</a></p> </content>

<author>

<name>David Dossett</name>

</author>

</entry>

<entry>

<title>Visual Studio Code September 2021</title>

<link href="https://code.visualstudio.com/updates/v1_61"/>

<updated>2021-10-07T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_61</id>

<content type="html"> <p>Visual Studio Code September 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_61">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>How We Made Bracket Pair Colorization 10,000x Faster In Visual Studio Code</title>

<link href="https://code.visualstudio.com/blogs/2021/09/29/bracket-pair-colorization"/>

<updated>2021-09-29T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/09/29/bracket-pair-colorization</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/09/29/bracket-pair-colorization">Read the full article</a></p> </content>

<author>

<name>Henning Dieterichs</name>

</author>

</entry>

<entry>

<title>Visual Studio Code August 2021</title>

<link href="https://code.visualstudio.com/updates/v1_60"/>

<updated>2021-09-02T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_60</id>

<content type="html"> <p>Visual Studio Code August 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_60">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Visual Studio Code July 2021</title>

<link href="https://code.visualstudio.com/updates/v1_59"/>

<updated>2021-08-05T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_59</id>

<content type="html"> <p>Visual Studio Code July 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_59">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>The Coming of Age of Notebooks</title>

<link href="https://code.visualstudio.com/blogs/2021/08/05/notebooks"/>

<updated>2021-08-05T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/08/05/notebooks</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/08/05/notebooks">Read the full article</a></p> </content>

<author>

<name>Chris Dias</name>

</author>

</entry>

<entry>

<title>Visual Studio Code June 2021</title>

<link href="https://code.visualstudio.com/updates/v1_58"/>

<updated>2021-07-08T17:00:00.000Z</updated>

<id>https://code.visualstudio.com/updates/v1_58</id>

<content type="html"> <p>Visual Studio Code June 2021</p> <p><a href="https://code.visualstudio.com/updates/v1_58">Read the full article</a></p> </content>

<author>

<name>Visual Studio Code Team</name>

</author>

</entry>

<entry>

<title>Workspace Trust in Visual Studio Code</title>

<link href="https://code.visualstudio.com/blogs/2021/07/06/workspace-trust"/>

<updated>2021-07-06T00:00:00.000Z</updated>

<id>https://code.visualstudio.com/blogs/2021/07/06/workspace-trust</id>

<content type="html"> <p></p> <p><a href="https://code.visualstudio.com/blogs/2021/07/06/workspace-trust">Read the full article</a></p> </content>

<author>

<name>Chris Dias</name>

</author>

</entry>

</feed>

**Comment by FahdSeddik** [comment]
Hey!
I believe you might find an answer in this [link](https://kb.blackbaud.com/knowledgebase/Article/47580). Hope this helps!
